velox/dwio/parquet/reader/ParquetReader.cpp

```diff
@@ -284,8 +286,20 @@ std::unique_ptr<ParquetTypeWithId> ReaderBase::getParquetColumnInfo(
       if (isFileColumnNamesReadAsLowerCase()) {
         folly::toLowerAscii(childName);
       }
-      auto childRequestedType =
-          requestedType ? requestedType->asRow().findChild(childName) : nullptr;
+
+      TypePtr childRequestedType = nullptr;
+      if (requestedType) {
+        if (requestedType->isRow()) {
+          auto fileTypeIdx =
+              requestedType->asRow().getChildIdxIfExists(childName);
```

Comment:
We probably want to add sanity checks: `VELOX_CHECK(requestedType->isRow())`

Reply:
I have already added the `requestedType->isRow()` check before, so do I need to add this one?